doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+This example is an extension of the `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ for a cylindrical geometry. The boundary condition and some parameters have been changed accordingly.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a solid-gas cylinder-shaped spouted bed
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-bed.prm``
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the spouting of spherical particles in air in a cylindrical geometry. As noted in the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-beml>`_, we use ``lethe-particles`` to fill the bed with particles, and ``lethe-fluid-particles`` as a solver of CFD-DEM.
+
+-------------------
+DEM Parameter File
+-------------------
+
+Here, we will focus only on the parts that have been modified.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a cylinder shaped spouted bed. We introduce the flow through a short channel that is connected to the inlet of the bed. A schematic image is shown below;
+
+.. image:: images/geometry.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+    :height: 10cm 
+
+The geometry of the bed was created using `Pointiwise <../../../tools/pointwise/pointowise.html>`_, and the overview of created mesh is also shown below;
+
+.. image:: images/mesh.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: mesh_ver
+    :height: 10cm
+
+In Unresolved-CFD-DEM, Cells need to have a enough volume to contain particles in it. Basically the size of cells should be least three time larger than the diameter of the particles in order to get stable calculation. So this can be expressed as follows.
+
+.. math:: 
+  \dfrac{d_p}{\Delta x} \leq 3
+
+where :math:`d_p` is the particle diameter, and :math:`\Delta x` is the characteristic size of the cell. In this example, we use particles with diameters of 5 mm which means that we need at least 15 mm for cell size. Also, we need to apply a relatively small mesh to the short channel below. Thus, we set the size of the central cylinder to 15mm, and made other grid coarser.
```

Comment:
```suggestion
where :math:`d_p` is the particle diameter, and :math:`\Delta x` is the characteristic size of the cell. In this example, we use particles with diameters of 5 mm which means that we need at least 15 mm for cell size. Also, we need to apply a relatively small mesh to the short channel below. Thus, we set the size of the central cylinder to 15mm, and made other grid coarser.
```
Use another symbol for Delta_x. It is not the cell size. You could use something larger. In fact, this is something we should revisit to ensure that when we use the cell size to calculate the void fraction using the cell size to guide the QCM we use the "Biggest possible size"